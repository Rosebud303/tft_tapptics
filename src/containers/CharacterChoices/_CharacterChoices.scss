.character-choices {
  height: 350px;
  background: red;
}