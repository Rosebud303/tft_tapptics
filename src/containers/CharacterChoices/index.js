import React, { Component } from 'react'

export default class CharacterChoices extends Component {
  render() {
    return (
      <div className='character-choices'>
        {/* this is going to be the area where function components will be mapped over for display */}
        <img src='https://i.imgur.com/g3EiCSi.png' />
        <img src='https://i.imgur.com/so58O3K.png' />
        <img src='https://i.imgur.com/4ByqI3E.png' />
        <img src='https://i.imgur.com/AmeGHvR.png' />
        <img src='https://i.imgur.com/3Fj3uOv.png' />
      </div>
    )
  }
}
