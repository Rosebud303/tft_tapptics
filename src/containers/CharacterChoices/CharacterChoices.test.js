import React from 'react';
import { shallow } from 'enzyme';
import CharacterChoices from './CharacterChoices';

describe('CharacterChoices', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CharacterChoices />)
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});