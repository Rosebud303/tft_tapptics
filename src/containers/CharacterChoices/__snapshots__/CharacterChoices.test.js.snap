// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterChoices should match the snapshot with all the data passed in 1`] = `
<div
  className="character-choices"
>
  <img
    src="https://i.imgur.com/g3EiCSi.png"
  />
  <img
    src="https://i.imgur.com/so58O3K.png"
  />
  <img
    src="https://i.imgur.com/4ByqI3E.png"
  />
  <img
    src="https://i.imgur.com/AmeGHvR.png"
  />
  <img
    src="https://i.imgur.com/3Fj3uOv.png"
  />
</div>
`;
